Assets/Scripts/NursePatrol.cs

```diff
@@ -60,9 +63,22 @@ void GotoNextPoint()
     // Update is called once per frame
     void Update()
     {
+
+        float distance = Vector3.Distance(transform.position, spy.transform.position);
+        // Chase spy
+        if (distance < agentChaseDistance && chase)
+        {
+            Vector3 dirToSpy = transform.position - spy.transform.position;
+            Vector3 newPos = transform.position - dirToSpy;
+
+            agent.SetDestination(newPos);
+        }
         // Choose the next destination point when the agent gets
         // close to the current one.
+
         if (!agent.pathPending && agent.remainingDistance < 0.5f)
             GotoNextPoint();
```

Comment:
Can we make this like a full if statement with curly braces? It can get messy if we ever want to add more logic and we forget this will only execute the line after.

Reply:
YAPP this was finnbarr's but i don't think he'll mind if i change it??

Reply:
yeah go for it

Reply:
Oh true! Yea let's change it cause it doesnt change the logic
also can we remove the whitespace below too